pkg/apis/monitoring/v1alpha1/types.go

```diff
@@ -72,6 +72,10 @@ type MonitoringStackSpec struct {
 	// +optional
 	// +kubebuilder:default={replicas: 2}
 	PrometheusConfig *PrometheusConfig `json:"prometheusConfig,omitempty"`
+
+	// Disables Alertmanager deployment
+	// +optional
+	DisableAlertmanager bool `json:"disableAlertmanager,omitempty"`
```

Comment:
I think it is better to have an `AlertManagerConfig` and have `Disabled` under it. So that the spec reads
```
spec:
  alertmanager:
    disabled: true
```

Reply:
I've thought about it and I think we can do it, but at the same time it seems like the only config option for the Alertmanager (but one never knows....).

Reply:
Yes, the idea is to namespace so that additional configs do not require changes to existing api.